lawnchair/src/app/lawnchair/ui/preferences/AppDrawerPreferences.kt

```diff
@@ -61,18 +55,34 @@ fun AppDrawerPreferences() {
             )
             SuggestionsPreference()
         }
+
+        PreferenceGroup(heading = stringResource(id = R.string.hidden_apps_label)) {
+            val hiddenApps = prefs2.hiddenApps.getAdapter().state.value
+            val showHiddenAppsInSearch = prefs2.showHiddenAppsInSearch.getAdapter()
+            val enableSmartHide = prefs2.enableSmartHide.getAdapter()
+            NavigationActionPreference(
+                label = stringResource(id = R.string.hidden_apps_label),
+                subtitle = resources.getQuantityString(R.plurals.apps_count, hiddenApps.size, hiddenApps.size),
+                destination = subRoute(name = AppDrawerRoutes.HIDDEN_APPS),
+            )
+            SwitchPreference(
+                label = stringResource(id = R.string.show_hidden_apps_search),
+                adapter = showHiddenAppsInSearch,
+            )
+            SwitchPreference(
+                label = stringResource(id = R.string.show_enable_smart_hide),
+                adapter = enableSmartHide,
+                enabled = !showHiddenAppsInSearch.state.value
+            )
```

Comment:
What are your thoughts of reversing the string name, e.g: "Hide in Search Results"? Therefore we could use `ExpandAndShrink` to the bottom switch, eg:

```
Hide in Search Results (off)
// Don't show Smart Hide
```
```
Hide in Search Results (on)
(ExpandAndShrink) Smart Hide
```

Note that if you implement this, you should `!` the adapter as seen in the "Show Search Bar" implementation:
```kotlin
            val showHiddenAppsInSearch = !prefs2.showHiddenAppsInSearch.getAdapter()
```

Therefore the full changes in this part should be:
```suggestion
            val hideHiddenAppsInSearch = !prefs2.showHiddenAppsInSearch.getAdapter()
            val enableSmartHide = prefs2.enableSmartHide.getAdapter()
            NavigationActionPreference(
                label = stringResource(id = R.string.hidden_apps_label),
                subtitle = resources.getQuantityString(R.plurals.apps_count, hiddenApps.size, hiddenApps.size),
                destination = subRoute(name = AppDrawerRoutes.HIDDEN_APPS),
            )
            SwitchPreference(
                label = stringResource(id = R.string.hide_hidden_apps_search),
                adapter = hideHiddenAppsInSearch,
            )
            ExpandAndShrink(hideHiddenAppsInSearch.state.value) {
                SwitchPreference(
                    label = stringResource(id = R.string.show_enable_smart_hide),
                    adapter = enableSmartHide,
                )
            }
```

(You should also change the name of the string in `/values/strings.xml`)